syntax = "proto3";

package chat;

service UserService {
  rpc Login (UserRequest) returns (UserReply);
  rpc ListUsers (Empty) returns (UserList);
}

message UserRequest {
  string username = 1;
}

message UserReply {
  string message = 1;
}

message Empty {}

message UserList {
  repeated string users = 1;
}
